docs/pages/management/admin/self-signed-certs.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Running Teleport with self-signed certs
+description: How run Teleport using self-certs
+---
+
+In this guide, we will show you how to run Teleport with self-signed certs, with remedies for problems caused
+by using self-signed certs.
+
+The Teleport Proxy Service authenticates itself to clients via TLS x509 certificates.
+If certificates are not configured for the Proxy Service, then it uses self-signed certificates,
+which clients will not trust by default. This can cause problems when using tools like
+`teleport`, `tsh`, or `tctl`.
+
+Additionally, when visiting the cluster's web portal, the certificate presented will not be trusted by your browser. In this case, you will likely see a page warning you that the site is not trusted.
+
+<Notice type="warning" title="DO NOT USE THIS IN PRODUCTION">
+  **DO NOT USE SELF-SIGNED CERTS IN PRODUCTION**
+
+  It is not recommended to run Teleport in production with self-signed certificates.
+  It is important to [properly configure certificates](#configuring-certificates-properly) from a trusted
+  [Certificate Authority](https://goteleport.com/blog/ssh-certificate-authorities-explained/)
+  to prevent a [Man-In-The-Middle (MITM)](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) attack.
+
+  This guide is provided for the benefit of users who are interested in evaluating Teleport in a
+  non-production environment, without having to configure certificates.
+</Notice>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
```

Comment:
Good point, and the Teleport Cloud tab doesnt make sense since in the context of this article either. I'll remove that include and make a custom one for this guide